columnflow/tasks/framework/parameters.py

```diff
@@ -4,13 +4,37 @@
 Custom luigi parameters.
 """
 
-from typing import Union
+from __future__ import annotations
+
+import copy
 
 import law
+import luigi
 
 from columnflow.util import test_float, DotDict
 
 
+class PlotFunctionParameter(luigi.Parameter):
+    """
+    Simply parameter subclass that provides a convenience method for copying an instance, assigning
+    a different default value and optionally changing the description text.
+    """
+
+    def with_default(
+        self,
+        default: str,
+        description: str | None = None,
+        amend_description: bool = False,
+    ) -> luigi.Parameter:
+        inst = copy.copy(self)
+        inst._default = default
+        if description is not None:
+            inst.description = description
+        elif amend_description:
+            inst.description += f"; default: {default}"
+        return inst
+
+
```

Comment:
This pattern for copying parameter instances could be generalized, too, if it proves effective.